packages/esm-patient-registration-app/src/config-schema.ts

```diff
@@ -12,7 +12,8 @@ export interface FieldDefinition {
   label?: string;
   uuid: string;
   placeholder?: string;
-  dateFormat?: string;
+  allowFutureDates?: boolean;
+  allowPastDates?: boolean;
```

Comment:
You have modified the type but you have not modified the actual config schema.

Reply:
You must modify the actual config schema; there should be a console warning about that visible in the browser console logs. Please use the browser console.

Reply:
Done